### DISCUSSION TITLE
Quick Comparison of ParameterValues?

### DISCUSSION THREAD

**Discussion by lappemic** [pybamm-team/PyBaMM#3958] [General]
Hello PyBaMM Team,

I'm working on a project that requires frequent comparisons between different sets of ParameterValues. Is there a built-in method or a recommended approach for quickly comparing ParameterValues to check if they are identical or to identify differences?
Thanks!

**Comment by valentinsulzer** [comment]
No built in method but you can just loop through the two ParameterValues, they are dictionary-like

**Reply by lappemic** [reply]
Thanks for getting back to me so quickly, @tinosulzer. I was wondering if there's another way to do this too.

Also, I have another question: How can I figure out which battery model is right for which use case? For example, if I want to model battery degradation, which model should I use? I saw the [Degradation experiments](https://github.com/pybamm-team/PyBaMM/blob/develop/docs/source/examples/notebooks/simulations_and_experiments/rpt-experiment.ipynb) notebook. Can i just use the Mohtat parameters therefore or also others?
